pyiron_contrib/tinybase/creator.py

```diff
@@ -0,0 +1,255 @@
+"""
+Creators of various things.
+
+They serve to give easy access to various classes, so that users do not have to
+manually import them.
+
+Rough spec:
```

Comment:
This is great, we should do more of this. I am really making an effort to at least do it at the class level, but it's really nice to see at the module level like this too!!!